packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```
@@ -3887,6 +3887,22 @@ describe('plugin-meetings', () => {
             }
           });
+
+          it('updates remote mute state when brb is enabled and audio is available', async () => {
+            meeting.audio = { handleServerRemoteMuteUpdate: sinon.stub() };
+
+            await meeting.beRightBack(true);
+
+            sinon.assert.calledOnceWithExactly(meeting.audio.handleServerRemoteMuteUpdate, meeting, true);
+          });
+
+          it.only('does not update remote mute state when brb is disabled and audio is available', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)